test/xspec.bats

```diff
@@ -1444,6 +1487,33 @@ load bats-helper
     [ "${lines[17]}" = "passed: 4 / pending: 0 / failed: 0 / total: 4" ]
 }
 
+@test "Saxon 11 and later: invoking xspec using SAXON_HOME finds Saxon jar and XML Catalog Resolver jar" {
+    if [ -z "${BUNDLED_XML_RESOLVER}" ]; then
+        skip "Saxon 11 and later only"
+    fi
+    export SAXON_HOME="${work_dir}/saxon ${RANDOM}"
+    mkdir "${SAXON_HOME}"
+    mkdir "${SAXON_HOME}/lib"
```

Comment:
These two `mkdir` lines can be just

```suggestion
    mkdir -p "${SAXON_HOME}/lib"
```

Reply:
Fixed in 7f34f8b2b728ce250d5bf7c3d5ea19bd8d78de80.